.github/workflows/pr-build.yml

```diff
@@ -77,6 +77,16 @@ jobs:
           # functional tests have a gap in them but actually building
           # real content exposes it.
 
+          # Setting this will mean that every live sample will be served
+          # like this:
+          #  <iframe src="//en-US/docs/Foo/_samples_/index.html">
+          # ...for example.
+          # Yes, it's potentially "insecure" because the iframe will execute
+          # whatever code is inserted into the code example. But since the
+          # whole (possible) domain for PR builds will never be somewhere
+          # where there are interesting cookies, it's a safe choice.
+          export BUILD_LIVE_SAMPLES_BASE_URL=/
```

Comment:
I really like this idea, but you'll have to modify the code at https://github.com/mdn/yari/blob/a2bd192b6065f9188b21f37f6dd6f38730a134af/kumascript/macros/LiveSampleURL.ejs#L20 because it throws an error when using `/`.

Reply:
I see now. 
<img width="1072" alt="Screen Shot 2020-09-01 at 10 05 05 PM" src="https://user-images.githubusercontent.com/26739/91923811-3fa5b100-ec9f-11ea-85a3-fd60429a0a70.png">

First of all, good catch!!

The saddest bug is that testing didn't capture this. But then again, how do you write a test that checks if a user has set a stupid env var value?
And the reason the PR Builds workflow didn't fail was that this change wasn't associated with an edit to a page that actually has an `EmbedLiveSample`.

I think the best solution is to make the macro smart enough to cope with the value NOT being a valid URL because that's pretty handy.

Reply:
Ok. I think I came up with a smart solution. If you set it to an empty string, it will use the document's URL. 
So the iframe will take the `url` from the document plus the `/_samples_/index.html` stuff. 
So that means, with this setting set to an empty string you end up with:

```html
<iframe 
  class="live-sample-frame sample-code-frame" 
  id="frame_Source_code_for_the_example" 
  frameborder="0" width="600" height="400" 
  src="/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index/_samples_/Source_code_for_the_example"></iframe>
```
when you're viewing http://localhost:5000/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index or http://localhost:5000/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index/

That also means that if you're building these pages and upload them to S3 and serve them as `https://pr123.dev.content.mdn.mozcloud.it` the `<iframes>` will work as they'll use that domain.